test/api/utils.ts

```diff
@@ -67,22 +71,31 @@ export function assertResultMatch(
 }
 
 /**
- * Check that the promise rejects with an expected error instance.
+ * Check that the promise rejects with an expected error.
  */
 export async function assertRejects(
   promise: PromiseLike<any>,
-  instanceOf: any
+  instanceOf: any,
+  message?: string | RegExp
 ) {
   try {
     await promise
     assert(false, 'Expected an error to be thrown')
   } catch (error) {
-    assert(error instanceof instanceOf)
+    assert(error instanceof instanceOf, error.message);
+    if (typeof message === 'string') {
+      assert.strictEqual(error.message, message);
+    } else if (message instanceof RegExp) {
+      assert(message.test(error.message));
+    }
   }
 }
 
 export function getAllPublicMethods(api: RippleAPI) {
-  return Object.keys(api).filter(key => !key.startsWith('_'))
```

Comment:
it turns out, `Object.keys()` on an object didn't return methods on its prototype (the RippleAPI base class that the instance inherits from)